internal/service/flexcluster/plural_data_source_schema.go

```diff
@@ -0,0 +1,31 @@
+package flexcluster
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-framework/datasource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+func DataSourcePluralSchema(ctx context.Context) schema.Schema {
+	return schema.Schema{
+		Attributes: map[string]schema.Attribute{
+			"project_id": schema.StringAttribute{
+				Required:            true,
+				MarkdownDescription: "Unique 24-hexadecimal digit string that identifies your project. Use the [/groups](#tag/Projects/operation/listProjects) endpoint to retrieve all projects to which the authenticated user has access.\n\n**NOTE**: Groups and projects are synonymous terms. Your group id is the same as your project id. For existing groups, your group/project id remains the same. The resource and corresponding endpoints use the term groups.",
+			},
+			"flex_clusters": schema.ListNestedAttribute{
```

Comment:
```suggestion
			"results": schema.ListNestedAttribute{
```
Would prefer to have consistency with other plural data sources in our provider, `flex_clusters` might also be redundant under the context of using `mongodbatlas_flex_clusters` data source

Reply:
I see that in resource policies it's like in here (`resource_policies`). I added a discussion topic for thursday about this so that we decide and unify the approach on all future resources

Reply:
Thanks, feel free to merge and we can adjust later if needed